packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -721,6 +721,11 @@ export const RemoteServerFactory = {
                     if (incomingRequest.path?.match(options.applySLASPrivateClientToEndpoints)) {
                         proxyRequest.setHeader('Authorization', `Basic ${encodedSlasCredentials}`)
                     }
+
+                    // /oauth2/trusted-agent/token endpoint requires a different auth header
+                    if (incomingRequest.path?.match(/\/oauth2\/trusted-agent\/token/)) {
+                        proxyRequest.setHeader('_sfdc_client_auth', encodedSlasCredentials)
+                    }
```

Comment:
we gotta keep in mind the documentation / release notes must include both versions for `commerce-sdk-react` and the `pwa-kit-runtime` for this feature to work.

Reply:
what happens if someone upgrades their `commerce-sdk-react` but not `pwa-kit-runtime`? Could we do an early exit and log errors "you must upgrade to pwa-kit-runtime@x.x.x to use TSOB"

Reply:
We designed the versioning strategy for `commerce-sdk-react` to be independent of the SDK versions, but it seems like it's no longer holding true, also for the private client feature.

We need to think about either decouple the dependency by exposing interfaces to each other, or change the versioning strategy...